README.md

```diff
@@ -213,10 +69,8 @@ errors. The [@sentry/rollup-plugin](#) is used to upload source maps.
 
 ## Known Limitations
 
-- Confluent Cloud connections require reauthenticating after 8 hours, and you will be prompted to
```

Comment:
I can see why we removed this from Known Limitations. Should we have a note elsewhere of expected session time?

Reply:
Can do in a follow up